plugins/modules/elb_target_group.py

```diff
@@ -866,19 +866,24 @@ def main():
         wait=dict(type='bool', default=False)
     )
 
-    module = AnsibleAWSModule(argument_spec=argument_spec,
-                              required_if=[
-                                  ['target_type', 'instance', ['protocol', 'port', 'vpc_id']],
-                                  ['target_type', 'ip', ['protocol', 'port', 'vpc_id']],
-                              ]
-                              )
+    state = module.params.get('state')
+    module_kwargs = {}
+    if state == 'present':
+        # ansible-collections/community.aws/issues/69
```

Comment:
These comments belongs to a changelog fragment. See https://github.com/ansible-collections/community.aws/tree/main/changelogs/fragments for examples.

Reply:
Done.